adapters/akka-http/src/main/scala/caliban/AkkaHttpAdapter.scala

```diff
@@ -113,24 +117,45 @@ trait AkkaHttpAdapter {
                   .forkDaemon
                   .flatMap(fiber => subscriptions.update(_.updated(message.id, fiber)))
               case other =>
-                sendMessage(sendTo, message.id, other, result.errors) *> IO.fromFuture(_ =>
-                  sendTo.offer(TextMessage(s"""{"type":"complete","id":"${message.id}"}"""))
-                )
+                sendMessage(sendTo, message.id, other, result.errors) *> IO
+                  .fromFuture(_ => sendTo.offer(TextMessage(s"""{"type":"complete","id":"${message.id}"}""")))
             }
       } yield ()
 
     get {
       extractUpgradeToWebSocket { upgrade =>
-        val (queue, source) = Source.queue[Message](0, OverflowStrategy.fail).preMaterialize()
-        val subscriptions   = runtime.unsafeRun(Ref.make(Map.empty[String, Fiber[Throwable, Unit]]))
+        val (queue: SourceQueueWithComplete[Message], source) =
```

Comment:
Do we need the explicit type annotation?

Reply:
No, I was trying to figure out what type of queue it was, forgot to take it out.